*** /home/runner/work/falco-missing-syscalls/falco-missing-syscalls/_output/falco_syscalls.txt	2021-03-25T22:15:33.687586+00:00
--- /home/runner/work/falco-missing-syscalls/falco-missing-syscalls/_output/kernel_next_syscalls.txt	2021-03-25T22:15:33.919587+00:00
***************
*** 1,14 ****
! Linux
! _llseek
! _newselect
  accept
  accept4
  access
  acct
  add_key
  adjtimex
  alarm
- arch_prctl
  bdflush
  bind
  bpf
--- 1,12 ----
! ##sname
  accept
  accept4
  access
  acct
  add_key
  adjtimex
+ adjtimex_time32
  alarm
  bdflush
  bind
  bpf
***************
*** 21,33 ****
--- 19,39 ----
  chown16
  chroot
  clock_adjtime
+ clock_adjtime32
  clock_getres
+ clock_getres_time32
  clock_gettime
+ clock_gettime32
  clock_nanosleep
+ clock_nanosleep_time32
  clock_settime
+ clock_settime32
  clone
+ clone3
  close
+ close_range
  connect
+ copy_file_range
  creat
  delete_module
  dup
***************
*** 37,52 ****
--- 43,63 ----
  epoll_create1
  epoll_ctl
  epoll_pwait
+ epoll_pwait2
  epoll_wait
  eventfd
  eventfd2
  execve
+ execveat
  exit
  exit_group
  faccessat
+ faccessat2
  fadvise64
+ fadvise64_64
  fallocate
  fanotify_init
+ fanotify_mark
  fchdir
  fchmod
  fchmodat
***************
*** 62,68 ****
--- 73,83 ----
  flock
  fork
  fremovexattr
+ fsconfig
  fsetxattr
+ fsmount
+ fsopen
+ fspick
  fstat
  fstat64
  fstatat64
***************
*** 70,94 ****
  fstatfs64
  fsync
  ftruncate
  futex
  futimesat
  get_robust_list
- get_thread_area
  getcpu
  getcwd
  getdents
  getdents64
  getegid
  getegid16
- getegid32
  geteuid
  geteuid16
- geteuid32
  getgid
  getgid16
- getgid32
  getgroups
  getgroups16
  getitimer
  getpeername
  getpgid
--- 85,110 ----
  fstatfs64
  fsync
  ftruncate
+ ftruncate64
  futex
+ futex_time32
  futimesat
+ futimesat_time32
+ get_mempolicy
  get_robust_list
  getcpu
  getcwd
  getdents
  getdents64
  getegid
  getegid16
  geteuid
  geteuid16
  getgid
  getgid16
  getgroups
  getgroups16
+ gethostname
  getitimer
  getpeername
  getpgid
***************
*** 99,108 ****
  getrandom
  getresgid
  getresgid16
- getresgid32
  getresuid
  getresuid16
- getresuid32
  getrlimit
  getrusage
  getsid
--- 115,122 ----
***************
*** 112,118 ****
  gettimeofday
  getuid
  getuid16
- getuid32
  getxattr
  init_module
  inotify_add_watch
--- 126,131 ----
***************
*** 122,136 ****
--- 135,161 ----
  io_cancel
  io_destroy
  io_getevents
+ io_getevents_time32
+ io_pgetevents
+ io_pgetevents_time32
  io_setup
  io_submit
+ io_uring_enter
+ io_uring_register
+ io_uring_setup
  ioctl
+ ioperm
  ioprio_get
  ioprio_set
  ipc
+ kcmp
+ kexec_file_load
  kexec_load
  keyctl
  kill
+ landlock_add_rule
+ landlock_create_ruleset
+ landlock_restrict_self
  lchown
  lchown16
  lgetxattr
***************
*** 140,168 ****
  listxattr
  llistxattr
  llseek
  lremovexattr
  lseek
  lsetxattr
  lstat
  lstat64
  madvise
  mincore
  mkdir
  mkdirat
  mknod
  mknodat
  mlock
  mlockall
- mmap
- mmap2
  mmap_pgoff
  mount
  mprotect
  mq_getsetattr
  mq_notify
  mq_open
  mq_timedreceive
  mq_timedsend
  mq_unlink
  mremap
  msgctl
--- 165,203 ----
  listxattr
  llistxattr
  llseek
+ lookup_dcookie
  lremovexattr
  lseek
  lsetxattr
  lstat
  lstat64
  madvise
+ mbind
+ membarrier
+ memfd_create
+ memfd_secret
+ migrate_pages
  mincore
  mkdir
  mkdirat
  mknod
  mknodat
  mlock
+ mlock2
  mlockall
  mmap_pgoff
  mount
+ mount_setattr
+ move_mount
+ move_pages
  mprotect
  mq_getsetattr
  mq_notify
  mq_open
  mq_timedreceive
+ mq_timedreceive_time32
  mq_timedsend
+ mq_timedsend_time32
  mq_unlink
  mremap
  msgctl
***************
*** 173,213 ****
--- 208,273 ----
  munlock
  munlockall
  munmap
+ name_to_handle_at
  nanosleep
+ nanosleep_time32
  newfstat
  newfstatat
  newlstat
  newstat
  newuname
+ ni_syscall
  nice
  old_getrlimit
  old_mmap
+ old_msgctl
  old_readdir
  old_select
+ old_semctl
+ old_shmctl
  oldumount
  olduname
  open
+ open_by_handle_at
+ open_tree
  openat
+ openat2
  pause
+ pciconfig_iobase
+ pciconfig_read
+ pciconfig_write
  perf_event_open
  personality
+ pidfd_getfd
+ pidfd_open
+ pidfd_send_signal
  pipe
  pipe2
  pivot_root
+ pkey_alloc
+ pkey_free
+ pkey_mprotect
  poll
  ppoll
+ ppoll_time32
  prctl
  pread64
  preadv
+ preadv2
  prlimit64
+ process_madvise
  process_vm_readv
  process_vm_writev
  pselect6
+ pselect6_time32
  ptrace
  pwrite64
  pwritev
+ pwritev2
  quotactl
+ quotactl_path
  read
+ readahead
  readlink
  readlinkat
  readv
***************
*** 215,220 ****
--- 275,281 ----
  recv
  recvfrom
  recvmmsg
+ recvmmsg_time32
  recvmsg
  remap_file_pages
  removexattr
***************
*** 224,244 ****
  request_key
  restart_syscall
  rmdir
  rt_sigaction
  rt_sigpending
  rt_sigprocmask
  rt_sigqueueinfo
- rt_sigreturn
  rt_sigsuspend
  rt_sigtimedwait
  rt_tgsigqueueinfo
  sched_get_priority_max
  sched_get_priority_min
  sched_getaffinity
  sched_getparam
  sched_getscheduler
  sched_rr_get_interval
  sched_setaffinity
  sched_setparam
  sched_setscheduler
  sched_yield
--- 285,309 ----
  request_key
  restart_syscall
  rmdir
+ rseq
  rt_sigaction
  rt_sigpending
  rt_sigprocmask
  rt_sigqueueinfo
  rt_sigsuspend
  rt_sigtimedwait
+ rt_sigtimedwait_time32
  rt_tgsigqueueinfo
  sched_get_priority_max
  sched_get_priority_min
  sched_getaffinity
+ sched_getattr
  sched_getparam
  sched_getscheduler
  sched_rr_get_interval
+ sched_rr_get_interval_time32
  sched_setaffinity
+ sched_setattr
  sched_setparam
  sched_setscheduler
  sched_yield
***************
*** 247,260 ****
  semctl
  semget
  semop
  send
  sendfile
  sendfile64
  sendmmsg
  sendmsg
  sendto
  set_robust_list
- set_thread_area
  set_tid_address
  setdomainname
  setfsgid
--- 312,327 ----
  semctl
  semget
  semop
+ semtimedop
+ semtimedop_time32
  send
  sendfile
  sendfile64
  sendmmsg
  sendmsg
  sendto
+ set_mempolicy
  set_robust_list
  set_tid_address
  setdomainname
  setfsgid
***************
*** 263,269 ****
  setfsuid16
  setgid
  setgid16
- setgid32
  setgroups
  setgroups16
  sethostname
--- 330,335 ----
***************
*** 275,284 ****
  setregid16
  setresgid
  setresgid16
- setresgid32
  setresuid
  setresuid16
- setresuid32
  setreuid
  setreuid16
  setrlimit
--- 341,348 ----
***************
*** 287,354 ****
  settimeofday
  setuid
  setuid16
- setuid32
  setxattr
  sgetmask
  shmat
- shmatcall
  shmctl
  shmdt
  shmget
  shutdown
  sigaltstack
  signal
  signalfd
  signalfd4
  sigpending
  sigprocmask
  socket
  socketcall
  socketpair
  splice
  ssetmask
  stat
  stat64
  statfs
  statfs64
  stime
  swapoff
  swapon
  symlink
  symlinkat
  sync
  syncfs
- sysctl
  sysfs
  sysinfo
  syslog
  tee
  tgkill
  time
  timer_create
  timer_delete
  timer_getoverrun
  timer_gettime
  timer_settime
  timerfd_create
  timerfd_gettime
  timerfd_settime
  times
  tkill
  truncate
! ugetrlimit
  umask
  umount
- umount2
  uname
  unlink
  unlinkat
  unshare
  uselib
  ustat
  utime
  utimensat
  utimes
  vfork
  vhangup
  vmsplice
--- 351,431 ----
  settimeofday
  setuid
  setuid16
  setxattr
  sgetmask
  shmat
  shmctl
  shmdt
  shmget
  shutdown
+ sigaction
  sigaltstack
  signal
  signalfd
  signalfd4
  sigpending
  sigprocmask
+ sigsuspend
  socket
  socketcall
  socketpair
  splice
+ spu_create
+ spu_run
  ssetmask
  stat
  stat64
  statfs
  statfs64
+ statx
  stime
+ stime32
  swapoff
  swapon
  symlink
  symlinkat
  sync
+ sync_file_range
+ sync_file_range2
  syncfs
  sysfs
  sysinfo
  syslog
  tee
  tgkill
  time
+ time32
  timer_create
  timer_delete
  timer_getoverrun
  timer_gettime
+ timer_gettime32
  timer_settime
+ timer_settime32
  timerfd_create
  timerfd_gettime
+ timerfd_gettime32
  timerfd_settime
+ timerfd_settime32
  times
  tkill
  truncate
! truncate64
  umask
  umount
  uname
  unlink
  unlinkat
  unshare
  uselib
+ userfaultfd
  ustat
  utime
+ utime32
  utimensat
+ utimensat_time32
  utimes
+ utimes_time32
  vfork
  vhangup
  vmsplice
